drafts/project-management-process.md

```diff
@@ -0,0 +1,340 @@
+
+Every project has a Project Lead, a Project Roadmap and a Status Board.
+
+### 📄 Goals
+
+At the heart of each Project are its Goals. They are the basic unit of work. A **Goal** is _an individual actionable unit of work that adds value to a release_. This includes issues reported by users, feature requests, bugs, Pull-Requests, general tasks such as refactoring, documentation or research. Goals may have parts or sub-tasks and they may have other Goals as dependencies.
+
+All of a Project's contributors work together to define, refine, collect, maintain and track Goals in the Pipeline but responsibility for this maintenance ultimately falls to the Project Lead and (if there is one) the Project Manager.
+
+**Attributes of a Goal:**
+
+A Goal description includes the following:
+- Name / Summary
+  - Describes what the goal is on high level
+  - Example 1: "Reduce the size of the go-ipfs executable by 30%"
+  - Example 2: "API Documentation for js-ipfs includes all public API methods"
+- Description
+  - What the problem is and how to achieve this goal
+  - Should tell you how you will know that the problem has been solved or the goal has been achieved
+  - Describe the type of this goal: is it a bug? a new feature? something else?
+- Status
+  - The status tells us which [stage](#pipelines-and-stages) the Goal has reached within the project's [workflow](#workflow).
+  - What is the current state of this goal? Is it to-be-reviewed? In progress? Blocked? Done?
+  - A goal can be in only one state at a time
+- Dependencies (where applicable)
+  - Links to other goals that must happen before or concurrently with this one
+- Any additional information that helps to reach this goal (optional)
+
+TODO: a clear example
+
+### 📘 Milestones
+
+A **Milestone** _groups together a set of goals that will achieve a significant, measurable improvement to the product or project._
+
+Milestones are the bridge between the [Status Board](#-status-boards) and the [Roadmap](#-project-roadmaps). The Roadmap focuses on the projected timeline while the the Status Board focuses on the current state of work in the Pipeline. Those contexts  each allow a different way of viewing the same pile of Goals. Milestones bridge these two contexts by gathering together related Goals and associating them with either an Estimated Date of Delivery or a Deadline so that we can situate those Goals within the Roadmap's timeline.
+
+A Milestone is completed when all Goals associated with it have been completed.
+
+**Attributes of a Milestone:**
+
+A Milestone has the following information:
+- Name
+  - Example 1: "Release go-ipfs v0.5.0"
+  - Example 2: "Working prototype of Pubsub implementation in go-ipfs"
+  - Example 3.1: "go-ipfs and js-ipfs implementations can interoperate"
+  - Example 3.2: "go-ipfs and js-ipfs clients use the same network"
```

Comment:
That particular section is directly from the original draft. @haadcode do you remember why the examples are written this way?